.changeset/changelog-stylelint.js

```diff
@@ -5,9 +5,42 @@
 const { getInfo, getInfoFromPullRequest } = require('@changesets/get-github-info');
 
 /**
- * @type {import('@changesets/types').ChangelogFunctions}
+ * @type {Array<string | undefined>}
+ */
+const CHANGESET_SECTIONS = 'Deprecated|Removed|Changed|Added|Fixed'.split('|');
```

Comment:
[suggest] How about refactoring the variable like this? If doing so, `@type {Array<string | undefined>}` also may be no longer needed.

```js
const CHANGESET_SECTIONS = ['Deprecated', 'Removed', 'Changed', 'Added', 'Fixed'];
```

Reply:
Working on it.